api/types/system_role.go

```diff
@@ -81,6 +81,10 @@ const (
 	// RoleAccessGraphPlugin is a role for Access Graph plugins to access
 	// Teleport's internal API and access graph.
 	RoleAccessGraphPlugin SystemRole = "AccessGraphPlugin"
+
+	// RoleAWSIdentityCenter is the role used by the AWS Identity Center integration
+	// when manipulating Teleport resources.
+	RoleAWSIdentityCenter SystemRole = "AWS-IdentityCenter"
```

Comment:
@tcsc Why do we need a system role? System roles are used by services that can be run as standalone agents (ssh, app, database, etc.), but the Identity Center integration always runs inside auth similar to hosted plugins?

Reply:
I used this role to allow the Identity Center integration to use the OIDC integration, as the list of roles that are allowed to use OIDC was small. 

That said, I drastically changed the way I used the OIDC integration since I originally did this. It may be a non-issue now. Will audit and get back to you.

Reply:
According to [`auth/integration/integrationv1`](https://github.com/gravitational/teleport/blob/d82ee8440cbc59d5f0b7d54b726e7e0b5053642b/lib/auth/integration/integrationv1/awsoidc.go#L46), the auth role should be fine.